src/app-layout/index.tsx

```diff
@@ -422,20 +398,23 @@ const OldAppLayout = React.forwardRef(
           )}
           <div className={clsx(styles.layout, disableBodyScroll && styles['layout-no-scroll'])}>
             {!navigationHide && (
-              <NavigationPanel
+              <Drawer
+                contentClassName={testutilStyles.navigation}
```

Comment:
why test-util styles are propagated via props and not defined inside the drawer?